google/cloud/apigee/registry/applications/v1alpha1/registry_styleguide.proto

```diff
@@ -0,0 +1,121 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+import "google/api/field_behavior.proto";
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryStyleGuideProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// StyleGuide defines a set of guidelines, rules, and linters that govern
+// the static analysis of API Specs.
+message StyleGuide {
+    // Resource name of the style guide.
+    string name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // A list containing style (format) descriptors for this style guide that 
+    // are specified as a Media Type (https://en.wikipedia.org/wiki/Media_type).
+    // Possible values include "application/vnd.apigee.proto", 
+    // "application/vnd.apigee.openapi", and "application/vnd.apigee.graphql",
+    // with possible suffixes representing compression types. These hypothetical
+    // names are defined in the vendor tree defined in RFC6838 
+    // (https://tools.ietf.org/html/rfc6838) and are not final.
+    repeated string mime_types = 2 [
+		(google.api.field_behavior) = REQUIRED
+	];
+
+    // A list of guidelines that are specified by this style guide.
+    repeated Guideline guidelines = 3;
+
+    // A list of linters that this style guide uses.
+    repeated Linter linters = 4;
+}
+
+message Guideline {
+    // Resource name of the guideline.
+    string name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // Human-meaningful name.
+    string display_name = 2 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // A detailed description of the guideline.
+    string description = 3;
+
+    // A list of rules that this guideline specifies.
+    repeated Rule rules = 4;
+
+    // Indicates the status of the guideline.
+    enum Status {
+        PROPOSED = 0;
+        ACTIVE = 1;
+        DEPRECATED = 2;
+        DISABLED = 3;
+    }
+    Status status = 5;
+}
+
+message Rule {
+    // Resource name of the rule.
+    string name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // Human-meaningful name.
+    string display_name = 2;
+
+    // A detailed description of the rule.
+    string description = 3;
+
+    // Name of the linter that enforces this rule.
+    string linter = 4 [
+		(google.api.field_behavior) = REQUIRED
+	];
+
+    // Name of the rule on the linter that enforces it.
+    string linter_rulename = 5 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // Severity of violating this rule.
+    enum Severity {
+        ERROR = 0;
```

Comment:
Is this a reasonable default value for the severity? [AIP-126](https://google.aip.dev/126#guidance) recommends having an "unspecified" default value if there isn't any clear "default" severity.

Reply:
As above, I agree that `SEVERITY_UNSPECIFIED = 0` would be a good first value.